pupil/views/check/question-virtual-keyboard.ejs

```diff
@@ -9,13 +9,7 @@ if (typeof nextQuestion === 'number') {
 
 <div class="hidden" id="js-content-div">
 
-    <div class="remaining-time">
-        Remaining time: <span id="js-page-timer" class="bold timer"></span>
-    </div>
-
-    <br style="clear:both"/>
-
-    <div class="question-container">
+    <div class="question-container" id="js-page-time-settings" data-value="<%= questionTimeLimit %>">
```

Comment:
I'm sure this will work, but I would have dropped the `questionTimeLimit` and `loadingTime` straight into the javascript with whatever values were correct for the test.  This seems a longer way round to me, and causes an additional dom lookup.  Is there a reason for it?